packages/web-react/src/components/Icon/demo/IconColorDualtone.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Grid } from '../../Grid';
+import { accentColors, emotionColors, textColors } from '../utils';
+import IconColorDemoFactory from './IconColorDemoFactory';
+
+const IconColorDualtone = () => {
+  const iconName = 'folder-dualtone';
+
+  return (
+    <Grid cols={{ desktop: 3, mobile: 1 }}>
```

Comment:
Mobile first, please.

```suggestion
    <Grid cols={{ mobile: 1, desktop: 3 }}>
```